packages/bpk-component-divided-card/README.md

```diff
@@ -0,0 +1,32 @@
+# bpk-component-divided-card
+
+> Backpack example component.
+
+## Installation
+
+Check the main [Readme](https://github.com/skyscanner/backpack#usage) for a complete installation guide.
+
+## Usage
+
+```js
+import React from 'react';
+import BpkDividedCard, { ORIENTATION } from '@skyscanner/backpack-web/bpk-component-divided-card';
```

Comment:
I would have seen this component not as a separate library all toegther but maybe actually inside the `bpk-component-card` and exported through there.

So would be an exported variant there instead, let me confirm with the Backpack team on how we are doing it for our other platforms